src/app/Http/Controllers/Operations/ListOperation.php

```diff
@@ -100,7 +100,7 @@ public function search()
                 // clear any past orderBy rules
                 $this->crud->query->getQuery()->orders = null;
                 // apply the current orderBy rules
-                $this->crud->query->orderByRaw($this->crud->model->getTable().'.'.$column['name'].' '.$column_direction);
+                $this->crud->query->orderByRaw(\DB::getTablePrefix().$this->crud->model->getTable().'.'.$column['name'].' '.$column_direction);
```

Comment:
Hmmm... I'm a bit skeptical about using DB here. About using DB anywhere, really. I'd rather we use Eloquent everywhere, to abstract away the DB layer. 

For example - **what happens here for MongoDB users**? MongoDB is not _officially supported_ by Backpack, but I know quite a few people use it with it...

Reply:
In CrudTrait we have this method called `getTableWithPrefix()` - take a look here - https://github.com/Laravel-Backpack/CRUD/blob/master/src/app/Models/Traits/HasRelationshipFields.php#L35-L46. 

Would this not be a good alternative to this @pxpm & @mgralikowski ? So instead of using `\DB::getTablePrefix()` or `$this->crud->model->getTable()` we use `$this->crud->model->getTableWithPrefix()`?